topics/sentinel.md

```diff
@@ -343,7 +343,7 @@ the network described above, but more likely possible with different
 layouts, or because of failures at the software layer), we have a similar
 issue as described in Example 2, with the difference that here we have
 no way to break the symmetry, since there is just a slave and master, so
-the master can't stop accepting queries when is disconnected by its master,
+the master can't stop accepting queries when is disconnected by its slave,
```

Comment:
I agree with you. Adding "it" is better.
Should I submit PR again? Or you'll update that?